less/index.less

```diff
@@ -4,6 +4,23 @@
    License: none (public domain)
 */
 
+// Variables
+
+@Maingray: #A9A9A9;
+@Green: olivedrab;
+
+// Mixins
+
+.flex-columnstart {
```

Comment:
Good job with you mixins.  Just a heads up, you can use your variables in your mixins as well.